knowledge-challenge-app/src/Components/StudentDashboard/StudentDashboard.js

```diff
@@ -177,19 +198,28 @@ function StudentDashboard(props) {
           </style>
           <Nav
             data-testid="navBar"
-            onSelect={(eventKey) => {
-              document.getElementById(eventKey).scrollIntoView({ behavior: "smooth" });
+            onSelect={(e) => {
+              document.getElementById(e);
```

Comment:
this is not recommended by react. Some consider it illegal. Much better to use state / conditional renderings.